### PR TITLE
[2.0.3] *: update release note

### PR DESCRIPTION
**Pull request by GMHDBJD**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-dm) that's required for repo owners to accept my contribution.

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v3.0 changes)
- [x] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: en/releases/2.0.3.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+- Fix the issue that DM-master becomes out of memory after DM is updated to v2.0.2 in the process of shard DDL coordination using the optimistic mode [#1643](https://github.com/pingcap/dm/pull/1643) [#1649](https://github.com/pingcap/dm/pull/1649)
+- Fix the issue that the source binding information is lost when DM is started for the first time after updated to v2.0.2 [#1649](https://github.com/pingcap/dm/pull/1649)
+- Fix the issue that the flag in the command `operate-source show -s` does not take effect [#1587](https://github.com/pingcap/dm/pull/1587)
+- Fix the issue that the command `operate-source stop <config-file>` fails because DM cannot connect to the source [#1587](https://github.com/pingcap/dm/pull/1587)
+- Fix the finer-grained issue that some migration errors might be wrongly ignored [#1599](https://github.com/pingcap/dm/pull/1599)
+- Fix the issue that the migration is interrupted when DM filters online DDL statements according to binlog event filtering rules that are configured [#1668](https://github.com/pingcap/dm/pull/1668)
+
+## Known issues
+
+[GitHub issues](https://github.com/pingcap/dm/issues?q=is%3Aissue+is%3Aopen+label%3Aaffected-v2.0.3)
```
Comment (lance6716) [inline en/releases/2.0.3.md:32]: we might remove `is:open`

Comment (lance6716) [reply en/releases/2.0.3.md:32]: (also in dm repo's changelog

**Review by CharLotteiu — APPROVED** [review]
LGTM

**Pull request merged**